CONTRIBUTING.md

```diff
@@ -59,9 +51,9 @@ poetry run pytest --cov --cov-config=.coveragerc --cov-fail-under=80 -n auto
 Once the necessary changes are made, run the following to generate the python code.
 
 ```
-poetry run nbdev_build_lib
-poetry run pre-commit run -a
-poetry run pre-commit run -a
+nbdev_build_lib
+pre-commit run -a
```

Comment:
I think we can remove references to `pre-commit` since we no longer require it. Black formatting (one reason we added `pre-commit` before) is still enforced anyway because our `settings.ini`  has `black_formatting` set to True.

Reply:
thanks butch, removed the `pre-commit` in contributing

and fixed the merge conflict so it should be `nbdev_export` now